.circleci/config.yml

```diff
@@ -50,3 +63,34 @@ jobs:
             - ~/.sbt
             - ~/.iv2/cache
             - ~/.m2
+      - persist_to_workspace:
+          root: ~/codesearch
+          paths:
+            - .dockerignore
+            - docker
+            - scripts
+            - Makefile
+            - codesearch-core.jar
+            - codesearch-server.jar
+
+  build_and_publish_docker_images:
+    working_directory: ~/codesearch
+    machine:
+      docker_layer_caching: true
+    steps:
+      - run:
+          docker login "$REGISTRY" -u "$DOCKER_USER" -p "$DOCKER_PASS"
```

Comment:
There's no point in specifying $REGISTRY because we're locked into Quay anyway – it's in our Makefile.